modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
```

Comment:
This is the first change for 6.0.3 so you'll need to bump the package.json version as well.